qa/tasks/rados.py

```diff
@@ -272,6 +272,13 @@ def thread():
                     )
                 tests[id_] = proc
             run.wait(tests.values())
+            if  config.get("wait_for_all_active_clean_pgs"):
+                # When thrashing OSDs rados test shouldn't remove pool
+                # before having all active+clean PGs and all OSDs up.
+                log.info('wait_for_all_active_clean_pgs') # Delete this later
+                manager.wait_for_clean()
+                log.info('wait_for_all_osds_up') # Delete this later
+                manager.wait_for_all_osds_up()
```

Comment:
Is there a reason to make this a flag?  I can't think of any tests where we'd expect the PGs not to go active+clean at the end of the test.

Reply:
Hi @athanatos , 
sorry for being so late at addressing this.
Basically I created this flag exclusively for `test_pool_min_size` because when there is a problem with PGs not being able to reach active state or etc (basically stuck in one of the wait_for_ tests) we would sometimes encounter a situation where rados tasks finishes first before the `wait_for_ tests` runs out of time, which results in the rados task deleting the pool and we would get out the `wait_for_ tests` with a passing because 0 PGs means it all the PGs are active. I didn't want to change how the `wait_for_ tests` works since some other tests might rely on this function, so I just created this flag exclusively for `test_pool_min_size`.